packages/aws-cdk-lib/aws-ses/lib/configuration-set.ts

```diff
@@ -153,6 +168,12 @@ export class ConfigurationSet extends Resource implements IConfigurationSet {
     });
 
     this.configurationSetName = configurationSet.ref;
+
+    this.configurationSetArn = this.stack.formatArn({
+      service: 'ses',
+      resource: 'configuration-set',
+      resourceName: this.configurationSetName,
+    });
```

Comment:
I think a getter method would be cleaner here, and would avoid having to duplicate this code in the `Import` class:

```ts
public get bootstrapQualifier(): string {
  return this.stack.formatArn({
    service: 'ses',
    resource: 'configuration-set',
    resourceName: this.configurationSetName,
  });
}

Reply:
@nmussy thanks for your comment. Not sure about your proposal, do you want me to implement a base class `ConfigurationSetBase` that both `Import` and `ConfigurationSet` will extend in order for them to both have this method?


I was originally inspired by `EmailIdentity` construct from `aws-ses` which has this duplicated code.
https://github.com/aws/aws-cdk/blob/31492c621c835e3fa08de2bbac24a0adc6fc2d69/packages/aws-cdk-lib/aws-ses/lib/email-identity.ts#L392-L396
https://github.com/aws/aws-cdk/blob/31492c621c835e3fa08de2bbac24a0adc6fc2d69/packages/aws-cdk-lib/aws-ses/lib/email-identity.ts#L500-L504

Do you have a specific implementation exemple in mind for your proposed solution that I could follow to update my PR?